src/helm/benchmark/presentation/summarize.py

```diff
@@ -955,19 +964,42 @@ def main():
         action="store_true",
         help="Skip write_run_display_json",
     )
+    # TODO: Remove this before release
+    parser.add_argument(
+        "--skip-aggregations-json",
+        action="store_true",
+        help="Skip write_executive_summary, write_runs, write_groups and write_cost_report",
+    )
+    # TODO: Remove this before release
+    parser.add_argument(
+        "--num-shards",
```

Comment:
What's a good value to set it at? Also, how are compute resources shared?

Reply:
Empirically, I found that 8 to 16 shards is good. There isn't really any kind of resource sharing logic; we're basically relying SLURM per-user quota limits to hold back jobs if needed.